### PR TITLE
Stabilitypage

### PR DESCRIPTION
**Pull request by MichaelCourtney**
https://github.com/freeorion/freeorion/issues/3961

Super rough. Just wanted to make a start

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6705,6 +6708,14 @@ ARTISTIC_SPECIES_TRAIT_TEXT
 
 List of Artistic species:'''
 
+INDEPENDENT_SPECIES_TITLE
+Independent Trait
+
+INDEPENDENT_SPECIES_TRAIT_TEXT
+'''Independent species are uneffected by stability modifiers based on distance from capitals or regional adminstration.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
'''Independent species' stability is unaffected by modifiers based on distance from capitals or regional administration.
```

**Comment by MichaelCourtney** [comment]
I'm not likely to finish this before the weekly build, unless you want a partially done version.

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -6344,9 +6344,31 @@ METER_HAPPINESS_VALUE_LABEL
...
+• Environment - Bonus or Penalty based on environmental preference of species. Set at gamestart in game rules via the MISC: Stability tab.
+• Planet Size - Bonus or Penalty based on planet size. Set at gamestart in game rules via the MISC: Stability tab.
+• Liked/Disliked Focus
+• Liked/Disliked Policies
+• Liked/Disliked Buildings (On Planet)
+• Liked/Disliked Buildings (In-system)
+• Liked/Disliked Buildings (Off-system)
+• Liked/Disliked Specials (On Planet)
+• Liked/Disliked Specials (In-system)
+• Liked/Disliked Specials (Off-system)
```
Comment (Grummel7) [inline default/stringtables/en.txt]: Liked/Disliked off-system specials do not affect stability, except for liked luxury resources when exported via capital market, which should be covered vi Policies.

For a complete list, World Tree is missing.

**Comment by MichaelCourtney** [comment]
Its maybe ok, but likely needs more work. I'm a little stumped as to where to go from here with this. Feedback welcome

**Pull request merged**